scripts/ubuntu/1-install-dependendcies.sh

```diff
@@ -29,32 +29,40 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get upgrade -y
 
 # TODO Comment on what dependency is required for:
 packagelist=(
-    apt-transport-https=2.4.11
-    avahi-utils=0.8-5ubuntu5.2                  # Matter uses Avahi
-    ca-certificates=20230311ubuntu0.22.04.1
-    docker-ce=5:24.0.7-1~ubuntu.22.04~jammy     # Test Harness uses Docker
-    figlet=2.2.5-3
-    g++=4:11.2.0-1ubuntu1
-    gcc=4:11.2.0-1ubuntu1
-    generate-ninja=0.0~git20220118.0725d78-1
-    libavahi-client-dev=0.8-5ubuntu5.2
-    libcairo2-dev=1.16.0-5ubuntu2
-    libdbus-1-dev=1.12.20-2ubuntu4.1
-    libgirepository1.0-dev=1.72.0-1
-    libglib2.0-dev=2.72.4-0ubuntu2.2
-    libreadline-dev=8.1.2-1
-    libssl-dev=3.0.2-0ubuntu1.14
-    net-tools=1.60+git20181103.0eebece-1ubuntu5
-    ninja-build=1.10.1-1
-    npm=8.5.1~ds-1
-    pkg-config=0.29.2-1ubuntu3
-    python3-pip=22.0.2+dfsg-1ubuntu0.4          # Test Harness CLI uses Python              
-    python3-venv=3.10.6-1~22.04                 # Test Harness CLI uses Python
-    software-properties-common=0.99.22.9
-    toilet=0.3-1.4
-    unzip=6.0-26ubuntu3.1
+    "apt-transport-https (=2.4.11)"
+    "avahi-utils (>=0.8-5ubuntu5.2)"                 # Matter uses Avahi
+    "ca-certificates (=20230311ubuntu0.22.04.1)"
+    "docker-ce (>=5:24.0.7-1~ubuntu.22.04~jammy)"    # Test Harness uses Docker
+    "figlet (=2.2.5-3)"
+    "g++ (=4:11.2.0-1ubuntu1)"
+    "gcc (=4:11.2.0-1ubuntu1)"
+    "generate-ninja (=0.0~git20220118.0725d78-1)"
+    "libavahi-client-dev (=0.8-5ubuntu5.2)"
+    "libcairo2-dev (=1.16.0-5ubuntu2)"
+    "libdbus-1-dev (=1.12.20-2ubuntu4.1)"
+    "libgirepository1.0-dev (=1.72.0-1)"
+    "libglib2.0-dev (=2.72.4-0ubuntu2.2)"
+    "libreadline-dev (=8.1.2-1)"
+    "libssl-dev (>=3.0.2-0ubuntu1.14)"               # Apparently with each update, previous versions of the library are removed
+    "net-tools (=1.60+git20181103.0eebece-1ubuntu5)"
+    "ninja-build (=1.10.1-1)"
+    "npm (=8.5.1~ds-1)"
+    "pkg-config (=0.29.2-1ubuntu3)"
+    "python3-pip (=22.0.2+dfsg-1ubuntu0.4)"          # Test Harness CLI uses Python              
+    "python3-venv (=3.10.6-1~22.04)"                 # Test Harness CLI uses Python
+    "software-properties-common (=0.99.22.9)"
+    "toilet (=0.3-1.4)"
+    "unzip (>=6.0-26ubuntu3.1)"
 )
-sudo DEBIAN_FRONTEND=noninteractive sudo apt-get install ${packagelist[@]} -y --allow-downgrades
+
+SAVEIFS=$IFS
+IFS=$(echo -en "\r")
+for package in ${packagelist[@]}; do
+  sudo DEBIAN_FRONTEND=noninteractive sudo apt satisfy ${package[@]} -y --allow-downgrades
+done
+IFS=$SAVEIFS 
```

Comment:
If we want to call 1 time once... We need a command like this:  `apt satisfy 'package1 (>=34.1)', package2 (=1.1)`

Packages must be separated by commas